libs/utils/sidekick.js

```diff
@@ -22,17 +27,37 @@ export default function init({ createTag, loadBlock, loadScript, loadStyle }) {
     getModal(null, { id: 'preflight', content, closeEvent: 'closeModal' });
   };
 
+  const addVersion = async () => {
+    const sk = document.querySelector('helix-sidekick');
+    const statusJson = JSON.parse(sk.getAttribute('status'));
+    const sourceUrl = statusJson?.edit?.url;
+    const sourceCode = sourceUrl?.match(/sourcedoc=([^&]+)/)[1];
+    const sourceId = decodeURIComponent(sourceCode);
+    const url = `https://adobe.sharepoint.com/sites/adobecom/_api/web/GetFileById('${sourceId}')`;
+
+    const options = getReqOptions({
+      method: 'POST',
+      accept: 'application/json; odata=nometadata',
+      contentType: 'application/json;odata=verbose',
+    });
+    await fetch(`${url}/Publish('Last Published version')`, { ...options, keepalive: true });
+  };
```

Comment:
I think the majority of this code should live in version-history, not in sidekick.js.

sidekick.js really should be a simple place to kick off the loading of some other feature. I don't think we want this much version specific code here.